README.md

```diff
@@ -273,13 +275,13 @@ Colored circles before the abbr mean the degree of recommendation:
 -  standard • 🟢 std
 -  standard input output • 🟢 stdio
 -  string • 🟢 str
--  synchronous • sync
+-  synchronous • 🟢 sync
 -  synchronize • 🟢 sync
```

Comment:
We need only the noun "synchronization", don't we?

Reply:
Yes, I had this commit on local and we discussed nouns after fixing it right now.